packages/core/src/config/config-resolvers.ts

```diff
@@ -71,6 +71,11 @@ export function resolvePlugins(
 ): Plugin[] {
   if (!plugins) return [];
 
+  if (isBrowser()) {
+    // TODO: show warning that plugins are not supported in browser
```

Comment:
Can we at least put `console.error` here?

Reply:
This is not an issue to put `logger.warn` but im not sure about the condition (we almost always have at least one default plugin, would it be correct to check for other plugins where `plugin.id !== ''` or just for presence of `plugin.id`) ?